### ISSUE TITLE
Localization missing from version 1.1.0 flatpak

### ISSUE THREAD

**Issue by sarmadka** [phw/peek#191]
Updating to 1.1.0 through flatpak resulted in missing Arabic localization (or maybe all localizations). The AppImage version still has the Arabic localization available. Running it through command line gave me this:
```
sarmad@sarmad-Oryx-Pro:~$ flatpak run com.uploadedlobster.peek 

(peek:3): Gtk-WARNING **: Locale not supported by C library.
	Using the fallback 'C' locale.

** (peek:3): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-qRHVXxe0zN: Connection refused
Using screen recorder backend gnome-shell
```

Trying a different locale:
```
sarmad@sarmad-Oryx-Pro:~$ LANG=ru flatpak run com.uploadedlobster.peek 

(peek:3): Gtk-WARNING **: Locale not supported by C library.
	Using the fallback 'C' locale.

** (peek:3): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-qRHVXxe0zN: Connection refused
Using screen recorder backend gnome-shell
```

This is on up-to-date Ubuntu Gnome 17.04. Gnome version is 3.24.2

**Comment by phw** [comment]
This should work, can you check your flatpak version and what `flatpak list -a` shows? It should show both `com.uploadedlobster.peek` and `com.uploadedlobster.peek.Locale`. If `com.uploadedlobster.peek.Locale` is missing, try removing and reinstalling `com.uploadedlobster.peek`. It should pull in the Locale package as well.

**Comment by phw** [comment]
Btw, the arabic translation is missing a single string that is marked as requiring review, could you take a look?

https://hosted.weblate.org/translate/peek/translations/ar/?type=fuzzy

Thanks in advance

**Comment by sarmadka** [comment]
Thanks @phw, uninstalling and reinstalling solved the issue.

I looked at the translation and it looks good but I suggested a minor change. Basically, the translation was saying "simple animated picture recorder..." and I suggested changing it into "simple screen recorder...". It's a very minor change, so if you prefer to stick with the original suggestion then it's perfectly fine; I just thought my suggestion is more descriptive of what the app actually does.

Thanks for the great app by the way :+1:
